Makefile

```diff
@@ -42,6 +44,10 @@ babelify: dist
 	@$(BABEL) lib/util --out-dir dist/lib/util
 	@$(BABEL) test --out-dir dist/test
 
+node_modules: package.json
```

Comment:
it's very weird because the files doesn't have this change: https://github.com/Automattic/wpcom.js/blob/update/babel-runtime/Makefile